router/dealer.go

```diff
@@ -692,43 +701,47 @@ func (d *dealer) cancel(caller *Session, msg *wamp.Cancel) {
 
 	// Cancel mode should be one of: "skip", "kill", "killnowait"
 	mode := wamp.OptionString(msg.Options, "mode")
```

Comment:
I can't recall where I saw that 'skip' should be the default (can't find it now), but I noticed that on the client, "killnowait" is the default when none is given.

Either way, it seemed that anything other than those 3 should be an error, except when omitted should be defaulted.